data/report_templates/project_summary.md

```diff
@@ -17,6 +17,11 @@ NGI Project Name
 NGI Project ID
 :   {{ project.ngi_id }}
 
+{% if project.ngi_facility %}
```

Comment:
Ah ha, it does created extra line breaks. I dint see the `.md` file at all. Just was seeing the `html` and seemed fine :sweat_smile: 

But also this fixed it, adding `-` to the end and after `%`
```
{% if project.ngi_facility -%}
NGI Facility
:   Genomics {{ project.ngi_facility }} Stockholm
{%- endif %}
```